components/common/src/test/java/com/hotels/styx/config/validator/DocumentFormatTest.java

```diff
@@ -171,6 +170,23 @@ public void checksIntegerFieldTypes() throws Exception {
         assertThat(result, is(true));
     }
 
+    @Test
+    public void convertsStringsToIntegerValues() throws Exception {
+        JsonNode rootObject = YAML_MAPPER.readTree(
```

Comment:
You can get the IntelliJ IDEA code formatter to align your concatenated strings properly if you start the with an empty string on the initial line, like:
```
 JsonNode rootObject = YAML_MAPPER.readTree(""
                + "root: \n"
                + "  myInt: '5' \n");
```
It's a bit weird, but will save you from wrestling with the formatter or getting misaligned code by accident.

Reply:
Ok. I applied this technique to all other tests in this file too.